openlineplanner-backend/src/main.rs

```diff
@@ -100,44 +96,33 @@ async fn main() -> std::io::Result<()> {
     #[rustfmt::skip]
     let config = Config::builder()
         .set_default("cache.dir", "./cache/").unwrap()
-        .set_default("data.dir", "./pbf/").unwrap()
+        .set_default("data.dir", "./data/").unwrap()
         .add_source(config::File::with_name("Config.toml").required(false))
         .build()
         .unwrap();
 
-    let (streets, buildings) = load_base_data(&config);
     let layers = load_layers(&config);
     let config = web::Data::new(config);
 
     log::info!("loading data done");
 
     HttpServer::new(move || {
         let cors = Cors::default()
-            .allowed_origin("https://openlineplanner.xatellite.space")
-            .allowed_origin("http://localhost:3000")
-            .allowed_origin_fn(|origin, _req_head| {
-                origin
-                    .as_bytes()
-                    .ends_with(b".openlineplanner.xatellite.space")
-            })
-            .allowed_origin_fn(|origin, _req_head| origin.as_bytes().ends_with(b"localhost:3000"))
-            .allowed_methods(vec!["GET", "POST", "DELETE"])
-            .allowed_headers(vec![http::header::AUTHORIZATION, http::header::ACCEPT])
-            .allowed_header(http::header::CONTENT_TYPE)
-            .max_age(3600);
+            .allow_any_origin()
```

Comment:
Lets make this more restrictive again.. but fixing it in a separate new issue.

Reply:
I think we can just straight up get rid of CORS except for local development

Reply:
True! We are now serving from the same origin right?